### PR TITLE
Item Sheet V2: Non-Activated Items

### PR DESCRIPTION
**Pull request by Fyorl**
image

### CODE REVIEW

File: module/applications/fields.mjs [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+ * @param {BooleanField} field               The field.
+ * @param {FormInputConfig<boolean>} config  The input configuration.
+ * @returns {HTMLElement}
+ */
+export function createCheckboxInput(field, config) {
+  const input = document.createElement("dnd5e-checkbox");
+  input.name = config.name;
+  if ( config.value ) input.checked = true;
+  foundry.applications.fields.setInputAttributes(input, config);
+  if ( "ariaLabel" in config ) input.arialLabel = config.ariaLabel;
```
Comment (krbz999) [inline module/applications/fields.mjs:12]:
```suggestion
  if ( "ariaLabel" in config ) input.ariaLabel = config.ariaLabel;
```

**Review by arbron — APPROVED** [review]
These changes are looking pretty nice. Noticed something in class description tab in dark mode I missed earlier:

Screenshot 2024-08-11 at 08 55 54

File: module/applications/fields.mjs (hunk 1 above)
Comment (Fyorl) [reply module/applications/fields.mjs]: Oops, thank you

**Comment by Fyorl** [comment]
> These changes are looking pretty nice. Noticed something in class description tab in dark mode I missed earlier:
> 
> Screenshot 2024-08-11 at 08 55 54

I hadn't included all the journal dark mode styles in this PR but I've added them now. It should look like this:

image

**Pull request merged**